.github/workflows/ci-backend.yml

```diff
@@ -197,22 +197,29 @@ jobs:
                   sudo apt-get update
                   sudo apt-get install libxml2-dev libxmlsec1-dev libxmlsec1-openssl
 
-            - name: Install python dependencies
-              run: |
-                  uv pip install --system -r requirements.txt -r requirements-dev.txt
+            # First running migrations from master, to simulate the real-world scenario
 
-            - uses: actions/checkout@v3
+            - name: Checkout master
+              uses: actions/checkout@v3
               with:
                   ref: master
 
-            - name: Run migrations up to master
+            - name: Install python dependencies for master
               run: |
-                  # We need to ensure we have requirements for the master branch
-                  # now also, so we can run migrations up to master.
                   uv pip install --system -r requirements.txt -r requirements-dev.txt
+
+            - name: Run migrations up to master
+              run: |
                   python manage.py migrate
 
-            - uses: actions/checkout@v3
+            # Now we can consider this PR's migrations
+
+            - name: Checkout this PR
+              uses: actions/checkout@v3
+
+            - name: Install python dependencies for this PR
```

Comment:
this also needs to run this PR's migrations, I don't think check migrations below does that

Reply:
Weirdly, it should, yet it looks like it never did. 🤔 Let's add that, sure